server/routers/deliverables.js

```diff
@@ -14,55 +17,66 @@ router.post('/add', (req, res) => {
     else if (description.length > 200) {
         res.status(400).send({ "message": "Description is too large" })
     }
-    else if(link ===null){
-        res.status(400).send({"message":"Link is invalid"})
+    else if (url === null) {
+        res.status(400).send({ "message": "Link is invalid" })
     }
-    //creating the object
-    let deliverable = {
-        projectId,
-        title,
-        description,
-        link
+    else {
+        const checkProject = await Project.findOne({ where: { id: project_id } })
+        if (checkProject === null) {
+            res.status(400).send({ "message": "Project not found" })
+        }
     }
 
-    //storing it into the database
+    try {
+        const del = await Deliverable.create({ title, description, url, project_id })
 
+        res.status(200).send({ "message": del.title + " was created." })
+    }
+    catch (err) {
+        console.log(err)
+        res.status(400).send({ "message": "Something bad happened" })
+    }
 
-    //response (will be a message after the db is made)
-    res.status(200).send(deliverable)
 })
 
-router.get('/get', (req, res) => {
-    projectId = req.body.projectId
-
-    //search in db
+router.get('/get', async (req, res) => {
+    id = req.body.id
```

Comment:
No body in GET. Please also preserve the order of the request, as you defined in your previous PR